auth/config.go

```diff
@@ -31,6 +31,22 @@ type ConfigYellowfin struct {
 	SourceAccessFilter bool
 }
 
+// Permission holds all of the details to create the dynamic permission list
```

Comment:
There should be a little blurb here (just in a free floating comment), describing why the dynamic permission system was created.

Reply:
Will do, thanks